cmd/proxy/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM golang:1.11-rc
+FROM golang:1.11
```

Comment:
I don't think `golang:1.11` is out on Dockerhub yet, we can do this in another PR or just make this PR a DO NOT MERGE one until it gets published: https://hub.docker.com/r/library/golang/tags/

Reply:
@marwan-at-work thanks for catching that. 
I've added WIP to the PR. 
I'll remove that once the image get uploaded. 

The PR[1] to do that was created 30 minutes ago, so I expect it to be merged and uploaded soon.

1. https://github.com/docker-library/official-images/pull/4779